_posts/2017-05-17-forbes.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Forbes
+sourceURL: https://www.youtube.com/watch?v=JmC0xkCMFCE
+image: /images/forbes.png
+tags:
+- News
+- Notifications
+- Service Worker
+- Offline
+
+---
+
+Forbes redesigned their mobile experience as a PWA resulting in **43% increase** in sessions per user, ad viewability up **20%**, and **100% more engagement**.
```

Comment:
A few thoughts on this:

* Are these "rules" on what should be bolded or not bolded written down somewhere?
* I find the numbers bolded on their own doesn't add meaning. *15%* What is the significance of the 15%?
* I've encountered some items that don't have a number. For example, "look at more products."

All that said, I don't care that much. Updating the site is already too time-consuming. I'll make these changes so that I can close the PR.